index.html

```
@@ -88,6 +88,12 @@ <h1 class="hdr-title">VoiceIsolate Pro <span class="hdr-ver">v20.0</span></h1>
         <button class="btn btn-preset" data-preset="music">Music</button>
         <button class="btn btn-preset" data-preset="broadcast">Broadcast</button>
         <button class="btn btn-preset" data-preset="restoration">Restoration</button>
+        <button class="btn btn-preset" data-preset="whisper">Whisper</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)